SEED-platform/seed
# Nginx brotli compression

#### What's this PR do?
- Configures Nginx to automatically serve the pre-compressed `.br` versions of static assets if available
- Enables on-the-fly brotli compression for responses (replacing gzip)
- Moves the 5 font css files to a block that can be compressed without loading them individually
- Updates all translation files

#### How should this be manually tested?
- Run SEED via Nginx locally and:
  - Check that the responses show `Content-Encoding: br`
  - attempting to access an html file within the node_modules directory results in downloading the file rather than serving it (e.g. `/static/node_modules/angular-dragula/index.html`)
  - attempting to access a static seed html file serves it as text/html (e.g. `/static/seed/partials/home.html`)
